CHANGES.md

```diff
@@ -10,6 +10,7 @@ Change Log
 * Improved support for polygon entities using `perPositionHeight`, including supporting vertical polygons.  This also improves KML compatibility. [#6791](https://github.com/AnalyticalGraphicsInc/cesium/pull/6791)
 * Added `Cartesian3.midpoint` to compute the midpoint between two `Cartesian3` positions [#6836](https://github.com/AnalyticalGraphicsInc/cesium/pull/6836)
 * Added `equalsEpsilon` methods to `OrthographicFrustum`, `PerspectiveFrustum`, `OrthographicOffCenterFrustum` and `PerspectiveOffCenterFrustum`.
+* Added support for glTF extension [KHR_materials_unlit](https://github.com/KhronosGroup/glTF/tree/master/extensions/2.0/Khronos/KHR_materials_unlit) [#6977](https://github.com/AnalyticalGraphicsInc/cesium/pull/6977).
```

Comment:
Looks like this isn't targeting master, as the changelog is 2 versions out of date as of today's release.

Reply:
Yeah I'm merging into https://github.com/AnalyticalGraphicsInc/cesium/pull/6805 since that PR moves `gltfPipeline` from `ThirdParty` into Cesium core.

Reply:
@OmarShehata Create a section for October's release and move your changes there. Thanks!

Reply:
Oh, I see the problem now. I moved it into a new section for October. Thanks for pointing this out!